ui/src/app/components/runtime-configuration-panel/disk-selector.tsx

```diff
@@ -52,48 +58,76 @@ export const DiskSelector = ({
           </StyledExternalLink>
         </FlexRow>
       </FlexColumn>
-      <FlexRow style={styles.diskRow}>
-        <RadioButton
-          name='standardDisk'
-          data-test-id='standard-disk-radio'
-          style={styles.diskRadio}
-          disabled={disabled}
-          onChange={() =>
-            onChange({
-              ...diskConfig,
-              detachable: false,
-              detachableType: null,
-              existingDiskName: null,
-            })
-          }
-          checked={!diskConfig.detachable}
-        />
-        <FlexColumn>
-          <label style={styles.diskLabel}>Standard disk</label>
-          <span>
-            A standard disk is created and deleted with your cloud environment.
-          </span>
-          {diskConfig.detachable || (
-            <DiskSizeSelector
-              idPrefix='standard'
-              diskSize={diskConfig.size}
-              disabled={disabled}
-              style={{ marginTop: '11px' }}
-              onChange={(size: number) =>
-                onChange(
-                  maybeWithExistingDiskName(
-                    {
-                      ...diskConfig,
-                      size,
-                    },
-                    existingDisk
+      {isComputeTypeStandard(computeType) && (
```

Comment:
It does, it was more for me. Since it was being used multiple places with computeType === ComputeType.Standard and computeType !== ComputeType.Standard, it was easier for me to just keep copying the same thing.